packages/blade/src/components/Card/CardSurface.web.tsx

```diff
@@ -1,17 +1,14 @@
 import styled from 'styled-components';
 import BaseBox from '~components/Box/BaseBox';
-import type { Elevation } from '~tokens/global';
-import { castWebType } from '~utils';
 
-const CardSurface = styled(BaseBox)<{ surfaceLevel: 2 | 3; elevation: keyof Elevation }>(
+const CardSurface = styled(BaseBox)<{ surfaceLevel: 2 | 3 }>(
   ({ surfaceLevel, elevation, theme }) => {
     const backgroundColor = theme.colors.surface.background[`level${surfaceLevel}`].lowContrast;
 
     return {
       width: '100%',
       display: 'flex',
       flexDirection: 'column',
-      boxShadow: castWebType(theme.elevation[elevation]),
```

Comment:
why did we remove this?

Reply:
We don't need this anymore because CardSurface is extending BaseBox and in BaseBox now we have elevation prop